accelerator/main.c

```diff
@@ -8,7 +9,18 @@ static ta_core_t ta_core;
 static ta_http_t ta_http;
 static logger_id_t logger_id;
 
+static void ta_stop(int signal) {
+  if (signal == SIGINT || signal == SIGTERM) {
```

Comment:
Once these signals are caught, is there any verbose message?

Reply:
Yes, after the signals are caught and executed `ta_http_stop`, then go to `cleanup` in main,
`Destroying TA configurations` will be printed out by logger.